File: Testing/Code for Testing/Early Basic Syntax Testing/in_operator/test_in_operator.py
from dataclasses import dataclass, field

def _ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class _ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def main():
    nums = [10, 20, 30, 40, 50]
    if (30 in nums):
        print('found 30')
    if (99 in nums):
        print('should not print')
    fruits = ['apple', 'banana', 'mango']
    if ('banana' in fruits):
        print('found banana')
    if ('grape' in fruits):
        print('should not print')
    return 0


if __name__ == "__main__":
    main()

File: Testing/Code for Testing/Early Basic Syntax Testing/in_operator/test_in_operator.js

function main() {
    let nums = [10, 20, 30, 40, 50];
    if (nums.includes(30)) {
        console.log("found 30");
    }
    if (nums.includes(99)) {
        console.log("should not print");
    }
    let fruits = ["apple", "banana", "mango"];
    if (fruits.includes("banana")) {
        console.log("found banana");
    }
    if (fruits.includes("grape")) {
        console.log("should not print");
    }
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
